Cargo.toml

```diff
@@ -11,3 +11,13 @@ authors = [
 license = "ISC"
 repository = "https://github.com/rust-scraper/ego-tree"
 readme = "README.md"
+
+[features]
+serde = ["dep:serde"]
+
+[dependencies]
+serde = { version = "1.0.209" , optional = true }
```

Comment:
IIRC there shouldn't be a space before the comma

Reply:
I love your nazi attitude

Reply:
And I am even from mobile!

Reply:
Fixed!